packages/@divvi/mobile/src/pincode/authentication.ts

```diff
@@ -133,7 +134,12 @@ function storePinWithBiometry(pin: string) {
     key: STORAGE_KEYS.PIN,
     value: pin,
     options: {
-      accessControl: Keychain.ACCESS_CONTROL.BIOMETRY_CURRENT_SET,
+      // BIOMETRY_CURRENT_SET not working as intended on Android
+      // https://github.com/oblador/react-native-keychain/issues/725
+      accessControl:
+        Platform.OS === 'ios'
+          ? Keychain.ACCESS_CONTROL.BIOMETRY_CURRENT_SET
+          : Keychain.ACCESS_CONTROL.BIOMETRY_ANY_OR_DEVICE_PASSCODE,
```

Comment:
Shoudn't it be this? Why would we want to allow device passcode for this case?
```suggestion
          : Keychain.ACCESS_CONTROL.BIOMETRY_ANY,
```